website/content/docs/commands/init.mdx

```diff
@@ -0,0 +1,58 @@
+---
+description: |
+  The `packer init` Packer command is used to download Packer plugin binaries.
+page_title: packer init - Commands
+sidebar_title: <tt>init</tt>
+---
+
+# `init` Command
+
+-> **Note:** Packer does not currently have the notion of a state.
+
+
+-> **Note:** Currently, the plugin getter can only fetch binaries from public
+projects on *Github*.
+
+-> **Note:** Downlading from *Github*, set the `PKR_GITHUB_API_TOKEN` env var in
```

Comment:
Should we provide a github link with details of how to have a github API token?

Reply:
Yup adding that now